x-pack/plugins/enterprise_search/public/applications/shared/indexing_status/indexing_status.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { EuiPanel, EuiSpacer } from '@elastic/eui';
+
+import { IndexingStatusContent } from './indexing_status_content';
+import { IndexingStatusErrors } from './indexing_status_errors';
+import { IndexingStatusFetcher } from './indexing_status_fetcher';
+
+import { IIndexingStatus } from '../types';
+
+export interface IIndexingStatusProps extends IIndexingStatus {
+  viewLinkPath: string;
+  itemId: string;
+  getItemDetailPath?(itemId: string): string;
+  getStatusPath(itemId: string, activeReindexJobId: number): string;
+  onComplete(numDocumentsWithErrors: number): void;
+  setGlobalIndexingStatus?(activeReindexJob: IIndexingStatus): void;
+}
+
+export const IndexingStatus: React.FC<IIndexingStatusProps> = (props) => {
+  const statusPanel = (percentComplete: number) => (
+    <EuiPanel paddingSize="l" hasShadow>
+      <IndexingStatusContent percentageComplete={percentComplete} />
+    </EuiPanel>
+  );
+
+  return (
+    <IndexingStatusFetcher {...props}>
+      {(percentageComplete, numDocumentsWithErrors) => (
+        <div className="c-stui-indexing-status-wrapper">
+          {percentageComplete < 100 && statusPanel(percentageComplete)}
```

Comment:
Very minor comment, but I don't see the advantage of pulling out `statusPanel` into a separate const since it's not repeated or anything - would suggest just leaving it inline

```suggestion
          {percentageComplete < 100 && (
            <EuiPanel paddingSize="l" hasShadow>
              <IndexingStatusContent percentageComplete={percentComplete} />
            </EuiPanel>
          )}
```
Also, do we just show nothing if `percentageComplete` is 100 with no errors? Like, we don't even show a completed bar?